tests/test_wire/test_errors.py

```diff
@@ -198,8 +198,10 @@ class T(m.Product):
         b = m.In(m.Clock)
 
     class Foo(m.Circuit):
-        io = m.IO(I=In(T), O=m.Out(m.Clock))
+        io = m.IO(I=In(T), O=m.Out(m.Clock), x=m.Out(m.Bit))
```

Comment:
actually, this is weird. we should be checking driven outputs for isdefinition too.

Reply:
oh wait, nevermind. with the removal of `def definition()` method, we actually don't check for this. this should really be fixed. I'll look into it since there's #929 also.